src/main/java/net/dv8tion/jda/api/events/guild/member/update/GuildMemberUpdateAvatarEvent.java

```diff
@@ -0,0 +1,54 @@
+package net.dv8tion.jda.api.events.guild.member.update;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Member;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Indicates that a {@link net.dv8tion.jda.api.entities.Member Member} updated their {@link net.dv8tion.jda.api.entities.Guild Guild} avatar.
+ *
+ * <p>Can be used to retrieve members who change their per guild avatar, triggering guild, the old avatar id and the new avatar id.
+ *
+ * <p>Identifier: {@code avatar}
+ *
+ * <h2>Requirements</h2>
+ *
+ * <p>This event requires the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_MEMBERS GUILD_MEMBERS} intent to be enabled.
+ * <br>{@link net.dv8tion.jda.api.JDABuilder#createDefault(String) createDefault(String)} and
+ * {@link net.dv8tion.jda.api.JDABuilder#createLight(String) createLight(String)} disable this by default!
+ *
+ * <p>Additionally, this event requires the {@link net.dv8tion.jda.api.utils.MemberCachePolicy MemberCachePolicy}
+ * to cache the updated members. Discord does not specifically tell us about the updates, but merely tells us the
+ * member was updated and gives us the updated member object. In order to fire a specific event like this we
+ * need to have the old member cached to compare against.
+ */
+public class GuildMemberUpdateAvatarEvent extends GenericGuildMemberUpdateEvent<String>
+{
+    public static final String IDENTIFIER = "avatar";
```

Comment:
Since the `UserUpdateAvatarEvent` also uses "avatar" as its identifier, wouldn't it be better to use something like `guild_avatar` to prevent possible confusion?

Reply:
I was under the impression that the identifier is supposed to match the API field.

Reply:
Not sure what or *if* there are specific requirements for the identifier String.
The only usage I saw so far was in the Generic Event class for the `toString()` method

Reply:
I've made this change, if it's wrong I'm sure Minn will leave a review.